cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -186,12 +186,12 @@
     'children': [{
         'titles': ['Deputy Associate Director'],
         'slug':   'principal-deputy-general-counsel',
-        'name':   'Vacant',
+        'name':   'Rich Lepley',
```

Comment:
This is very minor, but the png version uses 'Richard Lepley' here. Should this be the same, or are both fine?